.gitlab-ci.yml

```diff
@@ -4,5 +4,4 @@ include:
 Unit Test:
   image: quay.io/ebi-ait/ingest-base-images:openjdk_11
   script:
-    - INGEST_API_ROOT=http://localhost:8080
```

Comment:
What does it use instead of localhost?

Reply:
Nothing, it's all mocked

Reply:
Defined [here](https://github.com/ebi-ait/ingest-state-tracking/blob/7b5376b28530baf35a3fb62355aa71859e40efd6/build.gradle#L26). It doesn't actually use the URL to make any requests as everything is mocked. I made it clear that it's a dummy